examples/springboot/base/kustomization.yaml

```diff
@@ -0,0 +1,4 @@
+resources:
+- deployment.yaml
+- service.yaml
```

Comment:
Please also add a configmapGenerator in the base as
```
configMapGenerator:
- files:
  - application.properties
  name: demo-configmap
```
And in the base directory, add a file `application.properties`
```
app.name=Kustomize Demo
```

Reply:
Thanks for your review. I will update this PR with the changes tonight.